tests/suites/0_stateless/13_tpch/13_0000_prepare.sh

```diff
@@ -98,9 +98,18 @@ echo "CREATE TABLE IF NOT EXISTS lineitem
     l_comment      STRING not null
 )" | $MYSQL_CLIENT_CONNECT
 
+
+#download data 
+echo "download start..."
+#curl -s -o ${CURDIR}/data/tpch.tar.gz  http://repo.databend.rs/dataset/stateful/tpch.tar.gz
+wget http://repo.databend.rs/dataset/stateful/tpch.tar.gz -d  -O ${CURDIR}/data/tpch.tar.gz
```

Comment:
I tested in my local is ok, but seems not to fetch the file down in CI
```
download start...
Setting --output-document (outputdocument) to /Users/runner/work/databend/databend/tests/suites/0_stateless/13_tpch/data/tpch.tar.gz
DEBUG output created by Wget 1.21.3 on darwin20.6.0.

/Users/runner/work/databend/databend/tests/suites/0_stateless/13_tpch/data/tpch.tar.gz: No such file or directory
```
@edPanda  
Any thought @everpcpc

Reply:
try adding `mkdir -p ${CURDIR}/data/` before wget